interfaces/PA-App/src/app/shared/password-toggle-input/password-toggle-input.component.ts

```diff
@@ -39,4 +48,22 @@ export class PasswordToggleInputComponent {
   toggleInputType() {
     this.inputType = this.isPassword() ? 'text' : 'password';
   }
+
+  private setValidity(state: boolean, emit = true) {
+    this.isValid = state;
+    if (emit) {
+      this.isValidChange.emit(state);
+    }
+  }
+
+  public async onChange() {
+    if (environment.isDebug) {
+      this.setValidity(true);
+      return;
+    }
+
+    const nativeInput = await this.passwordInput.getInputElement();
+    const nativeIsValid = nativeInput.checkValidity();
```

Comment:
I'll look into using the 'default' Angular-way of checking if a field is 'valid' or not, because this still feels weird that we have to do this 'by hand'...